### PR TITLE
Move BTC setup to JS

### PR DESCRIPTION
**Pull request by D4nte**
First step for #475, move all blockchain interactions to JS.

### CODE REVIEW

File: api_tests/e2e/rfc003/btc_eth/test.js [hunk 1]
```diff
@@ -24,12 +24,14 @@ const logger = test_lib.logger();
...
         this.timeout(5000);
+        await test_lib.btc_activate_segwit();
+        test_lib.btc_enable_generate_every(1000);
         await bob.wallet.fund_eth(bob_initial_eth);
         await alice.wallet.fund_eth(alice_initial_eth);
         await alice.wallet.fund_btc(10);
-        await test_lib.import_address(bob_final_address); // Watch only import
-        await test_lib.import_address(alice.wallet.btc_identity().address); // Watch only import
-        await test_lib.import_address(bob.wallet.btc_identity().address); // Watch only import
+        await test_lib.btc_import_address(bob_final_address); // Watch only import
```
Comment (LLFourn) [inline api_tests/e2e/rfc003/btc_eth/test.js]: ℹ️ I made a ticket about removing this: https://github.com/comit-network/comit-rs/issues/566

File: api_tests/test_lib.js [hunk 2]
```diff
@@ -22,10 +22,14 @@ module.exports.logger = function () {
...
 };
 
-module.exports.sleep = time => {
+async function sleep (time) {
     return new Promise((res, rej) => {
         setTimeout(res, time);
     });
+}
+
+module.exports.sleep = time => {
```
Comment (LLFourn) [inline api_tests/test_lib.js]: why not just set this `= sleep` ?

**Review by LLFourn — APPROVED** [review]

File: api_tests/test_lib.js (hunk 2 above)
Comment (D4nte) [reply api_tests/test_lib.js]: I did not know it was an option :)

Comment (D4nte) [reply api_tests/test_lib.js]: I fixed that locally and will push with a new PR.

**Pull request merged**